packages/round-manager/src/context/ApplicationContext.tsx

```diff
@@ -1,12 +1,15 @@
 import React, { createContext, useContext, useEffect, useReducer } from "react";
 import { GrantApplication } from "../features/api/types";
 import { useWallet } from "../features/common/Auth";
-import { getApplicationById } from "../features/api/application";
+import { getApplicationById, getApplicationsByRoundId } from "../features/api/application";
+import { Web3Provider } from "@ethersproject/providers";
 
 enum ActionType {
   SET_APPLICATIONS = "SET_APPLICATIONS",
```

Comment:
super nitpick, but if `SET_APPLICATIONS` is only going to set a single `application` in state, then could we change it to `SET_APPLICATION` (singular)?

Reply:
fixed in [ab9d830](https://github.com/gitcoinco/grants-round/pull/416/commits/ab9d8304102d22e01ea8b3094d05c07fed093e47)